components/UnifiedDocFeed/FeedOrderingDropdown.tsx

```diff
@@ -0,0 +1,166 @@
+import { sortOpts } from "./constants/UnifiedDocFilters";
+import { scopeOptions } from "~/config/utils/options";
+import { css, StyleSheet } from "aphrodite";
+import colors from "~/config/themes/colors";
+import { useEffect, useRef, useState } from "react";
+import icons from "~/config/themes/icons";
+
+function FeedOrderingDropdown({
+  selectedOrderingValue,
+  selectedScopeValue,
+  onOrderingSelect,
+  onScopeSelect,
+}) {
+  const [isOpen, setIsOpen] = useState(false);
+  const selectedOrderingObj = sortOpts.find(
+    (o) => o.value === selectedOrderingValue
+  );
+  const dropdownRef = useRef<HTMLDivElement>(null);
+
+  const _handleOrderingClick = (selected) => {
+    onOrderingSelect(selected);
+    if (selected.disableScope) {
+      setIsOpen(false);
+    }
```

Comment:
```suggestion
      setIsOpen(selected.disableScope);
 ```

Reply:
I decided to keep things as is because the right line would actually be

```
 setIsOpen(!selected.disableScope);
 ```
 
 instead of 
```
 setIsOpen(selected.disableScope);
 ```

Even though it is an extra two lines, having the if statement there makes it a lot easier for me to read.
My brain takes a while figuring out negation and how it relates to an op like `setIsOpen`